File: src/Models/Package.cs
namespace Client.Models
{
    public class Package
    {
        public string Name { get; set; }
        public string SourceA { get; set; }
        public string SourceAVersion { get; set; }
        public string SourceB { get; set; }
        public string SourceBVersion { get; set; }
        public bool ComparePrerelease { get; set; }
    }
}

File: src/src/models/package.ts
export class Package {
    isFetching!: boolean;
    compareFailedString!: string;
    name: string;
    fetchDate!: Date;
    sourceAVersion!: string;
    sourceBVersion!: string;
    sourceAUrl!:string;
    sourceBUrl!:string;
    isLatest!:boolean;
    comparePrerelease!:boolean;

    constructor(name:string) {
        this.name = name;
    }
}